content/en/registry/exporter-dotnet-otlp.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OTLP Exporter
+registryType: exporter
+isThirdParty: false
+language: dotnet
+tags:
+  - .NET
+  - C#
+  - dotnet
+  - exporter
+repo: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol
+license: Apache 2.0
+description: This library allows to export data to the OpenTelemetry Collector using the OpenTelemetry Protocol.
```

Comment:
I noticed this text is used in all other "OTLP exporter entries". But it's not completely correct, as OTLP exporters also allow sending data to other places outside the collector (e.g., directly to an o11y back-end).

Maybe something like this:
> This library allows to export telemetry data in the OpenTelemetry Protocol (OTLP) format.

I can modify all the other exporters to have the same text, once we agree on the description.

Reply:
agreed, we had similar statements across the docs and changed that there as well.

> This library allows to export telemetry data in the OpenTelemetry Protocol (OTLP) format.

I am good with that, but maybe we can also add:

> This library allows to export telemetry data in the OpenTelemetry Protocol (OTLP) format to the OpenTelemetry Collector and OTLP-compliant backends

Reply:
Nice! We could swap `backends` to `receivers`, both are fine with me. I opened an issue for it, I guess we could move on with this PR as is, then I can fix all the others in one go. 

https://github.com/open-telemetry/opentelemetry.io/issues/1793